workflows/epitopeprediction.nf

```diff
@@ -148,7 +148,7 @@ workflow EPITOPEPREDICTION {
         ],
         netmhcpan: [
             version      : "4.0",
-            software_md5 : "94aa60f3dfd9752881c64878500e58f3",
+            software_md5 : "f576cd2d1906d5fafd182d71e8caf886",
```

Comment:
I think this is the checksum of the `tar` file but we need the checksum of the `tar.gz` file.

Reply:
right, apparently safari automatically did that 🙄 disabled that now